projects/kubernetes-sigs/cluster-api/patches/0001-Adding-capi-support-for-Bottlerocket.patch

```diff
@@ -1,4 +1,4 @@
-From c0d89ec2a34390253ed684c6f375e7c98309db66 Mon Sep 17 00:00:00 2001
+From 10dd70d5cf1eaf47cd2060d3e83847058072af59 Mon Sep 17 00:00:00 2001
 From: Vignesh Goutham Ganesh <vgg@amazon.com>
 Date: Fri, 11 Jun 2021 10:43:09 -0700
 Subject: [PATCH 01/34] Adding capi support for Bottlerocket
```

Comment:
```suggestion
Subject: [PATCH 01/35] Adding capi support for Bottlerocket
```

same for all

Reply:
why does it need to change to 35? I only see 34 patches. Also, this is auto-generated.